app/src/main/res/layout/session_list_private_empty.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm guessing this file will be moved to a more permanent place once it exists?

Reply:
Good catch on this, this file is actually unnecessary.